### PR TITLE
Escape values in queries from PostgresCursor

### PR DESCRIPTION
**Pull request by lundibundi**
Modifies SelectBuilder to use escaping in generated queries.
toString() is changed to build() as that's more appropriate name.

### CODE REVIEW

File: lib/pg.utils.js [hunk 1]
```diff
@@ -103,6 +103,17 @@ const validateIdentifier = (name, type, prefix = '') => {
...
 };
 
+const generateQueryParams = (count, start = 1) => {
+  let params = '';
+  for (let i = start; i < start + count; i++) {
+    if (i !== start) {
+      params += ', ';
+    }
+    params += `$${i}`;
+  }
```
Comment (tshemsedinov) [inline lib/pg.utils.js]: ```params += (i === start ? '' : ', ') + `$${i}`;```

Comment (lundibundi) [reply lib/pg.utils.js]: I don't like using ternary when one of the branches of missing as it usually makes the statement more confusing ever though reducing line count.
Also, this function is a copy paste from @belochub pg provider PR.

Comment (tshemsedinov) [reply lib/pg.utils.js]: Ok

File: lib/.tern-port [hunk 2]
```diff
@@ -0,0 +1 @@
+33611
```
Comment (belochub) [inline lib/.tern-port]: Remove this?

**Pull request merged**